src/content/docs/ja/guides/imports.mdx

```diff
@@ -144,7 +223,7 @@ Astroは、ブラウザの[`WebAssembly`](https://developer.mozilla.org/ja/docs/
 
 ## Nodeビルトイン
 
-Astroのユーザーには、可能な限りNode.jsのビルトイン（`fs`、`path`など）を避けることをおすすめします。Astroは、将来的に複数のJavaScriptランタイムと互換性を持つことを目指しています。これには、`fs`などのNode組み込みモジュールをサポートしない[Deno](https://deno.land/)や[Cloudflare Workers](https://workers.cloudflare.com/)が含まれます。
+Astroのユーザーには、可能な限りNode.jsのビルトイン（`fs`、`path`など）を避けることをおすすめします。Astroは、アダプタを使用して複数のランタイムと互換性があります。これには、`fs`などのNodeビルトインモジュールをサポートしない[Deno](https://deno.land/)や[Cloudflare Workers](https://workers.cloudflare.com/)が含まれます。
```

Comment:
This part was recently updated in #2931, after updating this LGTM!

Reply:
@Yan-Thomas Thanks for your review! I missed the fix for the links. I have just fixed it.